### PR TITLE
feat(migrate):migrate history uniqueIdentifier in collaboration_project

### PR DESCRIPTION
**Pull request by PeachThinking**
#### What type of PR is this?
type-feature

#### What this PR does / why we need it:
migrate history uniqueIdentifier in collaboration_project
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-migrate/src/main/resources/migrate/common/V_4_3_0_4__add_unique_identifier.sql [hunk 1]
```diff
@@ -1 +1 @@
-ALTER TABLE `collaboration_project` ADD COLUMN `unique_identifier` VARCHAR(128) DEFAULT NULL COMMENT 'Unique identifier, from external system';
\ No newline at end of file
+ALTER TABLE `collaboration_project` ADD COLUMN `unique_identifier` VARCHAR(128) NOT NULL COMMENT 'Unique identifier, may from external system or UUID generated by ODC if no external identifier';
```
Comment (yizhouxw) [inline server/odc-migrate/src/main/resources/migrate/common/V_4_3_0_4__add_unique_identifier.sql:1]: how about simpliy use 

`update collaboration_project set unique_identifier=concat('ODC_', uuid()) where unique_identifier is null;`

Comment (PeachThinking) [reply server/odc-migrate/src/main/resources/migrate/common/V_4_3_0_4__add_unique_identifier.sql]: ok~

**Review by yhilmare — APPROVED** [review]
lgtm

**Review by yizhouxw — APPROVED** [review]
lgtm

**Review by smallsheeeep — APPROVED** [review]
LGTM

**Pull request merged**